services/ux-backend/main.go

```diff
@@ -94,3 +106,22 @@ func main() {
 	log.Fatal(err)
 
 }
+
+func newClient() (client.Client, error) {
```

Comment:
Please be more specific with your function names
```suggestion
func newK8sClient() (client.Client, error) {
```

Reply:
ack